modules/appeals_api/spec/requests/v2/legacy_appeals_controller_spec.rb

```diff
@@ -159,43 +159,6 @@
         expect(JSON.parse(response.body)).to eq body
       end
     end
-
-    context 'using the versioned namespace route' do
```

Comment:
This example is no longer relevant because the v0 controller doesn't inherit from the decision reviews controller anymore